code/__defines/colors.dm

```diff
@@ -0,0 +1,114 @@
+// BYOND lower-cases color values, and thus we do so as well to ensure atom.color == COLOR_X will work correctly
```

Comment:
Redefine much (like all of them) values defined in `code/defines/colors.dm`
Remove one file

Reply:
Files match, i deleted one